tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -163,6 +163,31 @@ SELECT ANY(CAST(2.34 AS DECIMAL(6, 2)))
 ----
 2.34
 
+
+## compare
+
+query IIIII
+select a > b, a < b, a = b, a <= b, a >= b from (select 3::Decimal(13,2) a , 3.1::Decimal(8,2) b);
```

Comment:
Can we auto cast integer to decimal in the underlying?

Reply:
Yes, check the following test.

Reply:
I can not find it like: where a = 0.3 or a> 2

Reply:
I see :)